modules/rosa-adding-tags-cli.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-adding-tags-cli{context}"]
+= Adding tags to a machine pool using the ROSA CLI
+
+You can add tags to a machine pool for your {product-title} cluster by using the ROSA CLI.
+
+.Prerequisites
+
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
```

Comment:
I have no problem with this as a sentence, but in a lot of other docs, we make each of the CLIs a separate line.